README.md

```diff
@@ -0,0 +1,58 @@
+# arns-react
+A React App for the Arweave Name System Registry, which lets users search for and purchase Names.
+
+## Getting Started
+
+### Running the app locally
```

Comment:
```suggestion
### Local Setup
```

Reply:
fixed in commit e4eff7bd90681427abf826291072d15215748323